client/components/site-stats-sticky-link/index.jsx

```diff
@@ -36,11 +39,23 @@ var SiteStatsStickyLink = React.createClass( {
 		}
 	},
 
+	preload: function() {
```

Comment:
After having a look at this PR, my initial suggestion would be:
- A wrapper inspired in Redux's `connect` that would expose `preload()` as a component prop, so as to minimize contact with the `sections` module. I may be overthinking, but my motivation is that IMO `sections` should be required in just one agreed-upon place.
- That, combined with something like `<Link>`, as @mtias suggests, which would be a "preload-connected" component:

```js
const Link = React.createClass( {
  propTypes: {
    preload: ….func.isRequired,
    preloadModule: ….string,
    children: …,
    href: …, title: …, onClick: …
  },
  render() {
    return (
      <a href={ … } onMouseEnter={ this.props.preload( this.props.preloadModule } …>
        { this.props.children }
      </a>
    );
  }
} );

export default connectToPreload( Link ); // for lack of a better name

// then used like so:
<Link
  href={ … }
  title={ … }
  preloadModule="foo" // optional
/>
```